### ISSUE TITLE
Failed to install it in R4.0

### ISSUE THREAD

**Issue by duo-zhao-sdu** [KasperSkytte/mmgenome2#33]
My system is Windows,and I have installed Rtools3.5 before the installation of mmgenome2.
After the first failure,I installed package Rtsne and backports bacause it told me that the auto installation didn't work.
What I have done doesn't solve the problem.
Now it shows:
> remotes::install_github("kasperskytte/mmgenome2")
Downloading GitHub repo kasperskytte/mmgenome2@master
These packages have more recent versions available.
It is recommended to update all of them.
Which would you like to update?

1: All                                       
2: CRAN packages only                        
3: None                                      
4: Rtsne     (0.15  -> 1d0f926d8...) [GitHub]
5: backports (1.1.6 -> 1.1.7       ) [CRAN]  
6: xfun      (0.13  -> 0.14        ) [CRAN]  

Enter one or more numbers, or an empty line to skip updates:
1
Rtsne     (0.15  -> 1d0f926d8...) [GitHub]
backports (1.1.6 -> 1.1.7       ) [CRAN]
tinytex   (NA    -> 0.23        ) [CRAN]
xfun      (0.13  -> 0.14        ) [CRAN]
modelr    (NA    -> 0.1.8       ) [CRAN]
Installing 4 packages: backports, tinytex, xfun, modelr
Installing packages into ‘C:/Users/ColdSymmetry/Documents/R/win-library/4.0’
(as ‘lib’ is unspecified)

  There are binary versions available but the source versions are later:
          binary source needs_compilation
backports  1.1.6  1.1.7              TRUE
tinytex     0.22   0.23             FALSE
xfun        0.13   0.14             FALSE
modelr     0.1.7  0.1.8             FALSE

installing the source packages ‘backports’, ‘tinytex’, ‘xfun’, ‘modelr’

trying URL 'https://cran.rstudio.com/src/contrib/backports_1.1.7.tar.gz'
Content type 'application/x-gzip' length 17676 bytes (17 KB)
downloaded 17 KB

trying URL 'https://cran.rstudio.com/src/contrib/tinytex_0.23.tar.gz'
Content type 'application/x-gzip' length 27142 bytes (26 KB)
downloaded 26 KB

trying URL 'https://cran.rstudio.com/src/contrib/xfun_0.14.tar.gz'
Content type 'application/x-gzip' length 69436 bytes (67 KB)
downloaded 67 KB

trying URL 'https://cran.rstudio.com/src/contrib/modelr_0.1.8.tar.gz'
Content type 'application/x-gzip' length 121333 bytes (118 KB)
downloaded 118 KB

* installing *source* package 'backports' ...
** 成功将'backports'程序包解包并MD5和检查(unpackaged 'backports' succesfully and MD5 checked)
** using staged installation
** libs

*** arch - i386
make: C:/PROGRA~1/R/R-40~1.0/etc/i386/Makeconf: No such file or directory
make: *** No rule to make target 'C:/PROGRA~1/R/R-40~1.0/etc/i386/Makeconf'.  Stop.
ERROR: compilation failed for package 'backports'
* removing 'C:/Users/ColdSymmetry/Documents/R/win-library/4.0/backports'
* restoring previous 'C:/Users/ColdSymmetry/Documents/R/win-library/4.0/backports'
Error: Failed to install 'unknown package' from GitHub:
  (converted from warning) installation of package ‘backports’ had non-zero exit status

**Comment by KasperSkytte** [comment]
Hi there. Seems to me like your R installation is troubled and not the mmgenome2 package itself. Can you try reinstalling R and try again? Or else try not updating the suggested packages. I haven't tested mmgenome2 with R 4.0.0 yet, but can do in the near future.

**Comment by KasperSkytte** [comment]
Oh, you need Rtools 4.0 as well. That's likely the cause, see https://mirrors.dotsrc.org/cran/bin/windows/Rtools/

**Comment by duo-zhao-sdu** [comment]
Thanks for your reply, I reinstalled R, Rtools and Rstudio then there comes another error shows:

> remotes::install_github("kasperskytte/mmgenome2")
Downloading GitHub repo kasperskytte/mmgenome2@master
KasperSkytte-mmgenome2-1c4663f/docs/reference/mmplot-1.png: truncated gzip input
tar.exe: Error exit delayed from previous errors.
Error: Failed to install 'unknown package' from GitHub:
  Does not appear to be an R package (no DESCRIPTION)

Then I downloaded the package and used command : 
>devtools::install_local("mmgenome2-master.zip") 
This solved my new question.Now I can use mmgenome2.
Thanks again.
